src/js/models/v2/account/BaseAccountSpendingRow.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * BaseAccountSpendingRow.js
+ * Created by James Lee 5/27/20
+ */
+
+import { formatMoney } from 'helpers/moneyFormatter';
+import { generatePercentage } from '../../../helpers/awardAmountHelper';
+
+const BaseAccountSpendingRow = {
+    populate(data) {
+        this.name = data.name || '';
+        this._grossOutlayAmount = data.gross_outlay_amount || 0;
+        this._obligatedAmount = data.obligated_amount || 0;
+        this._totalObligatedAmount = data.total_obligated_amount || 0;
+    },
+    get grossOutlayAmount() {
+        return formatMoney(this._grossOutlayAmount);
+    },
+    get obligatedAmount() {
+        return formatMoney(this._obligatedAmount);
+    },
+    get percentOfTotalObligations() {
+        if (this._totalObligatedAmount > 0) {
+            const percentage = generatePercentage(this._obligatedAmount / this._totalObligatedAmount);
+            if (percentage === ('0.00%')) return 'Less than 0.01%';
```

Comment:
curious as to why we have the parens around the `('0.00%')`?

Reply:
No reason, I removed it.